app/build.gradle

```diff
@@ -1,30 +1,15 @@
-/*
```

Comment:
can we leave the license header in? and for subsequent files too

Reply:
Looks like we accidentaly merged this without a license